tsconfig.json

```diff
@@ -26,73 +26,6 @@
     "./public/content",
     "./src/intl",
     "./src/data", // TODO: move to /public?
```

Comment:
We use `.ts` files in the `./src/data` folder, but currently we're not type-checking them... imo we should remove this as well and make sure those compile appropriately. Can be a separate task if preferred.

Reply:
I can do that in here too!

Reply:
Left a comment about this.

> We use .ts files in the ./src/data folder, but currently we're not type-checking them

@wackerow I don't think this is true. TS is checking those files anyway because those files are being imported by other modules. You can read more about the `exclude` option [here](https://www.typescriptlang.org/tsconfig#exclude).

I ran a test and I can confirm that TS is checking the `.ts` and `.tsx` files under `/data`.